### DISCUSSION TITLE
login issue

### DISCUSSION THREAD

**Discussion by Thanoon12k** [vercel/community#1812] [Help] [answered]
### Summary

i signup using my github but when ask me for my  mobile i write my mobile number 
here
Please enter your phone number to verify your login. You will receive a single use code to submit.

when press continue i got this error lable
Error:An unexpected internal error occurred
the code never sent to me

**Comment by swarnava** [comment, marked as answer]
Hi, please write us at registration@vercel.com

**Reply by Thanoon12k** [reply]
i'll do thanks for reply
